internal/service/rds/cluster.go

```diff
@@ -761,6 +761,10 @@ func resourceClusterCreate(ctx context.Context, d *schema.ResourceData, meta int
 			requiresModifyDbCluster = true
 		}
 
+		if v, ok := d.GetOkExists(names.AttrStorageType); ok {
+			input.StorageType = aws.String(v.(string))
+		}
```

Comment:
Why not

```suggestion
		if v, ok := d.GetOk(names.AttrStorageType); ok {
			input.StorageType = aws.String(v.(string))
		}
```

?

Reply:
I copied this from the logic in the final `else` block of the create method (a cluster not restored from anything). I believe `GetOk` is fine in this context though - I'll swap it out everywhere.